addons/mass_mailing/static/src/js/MassMailingWysiwyg.js

```diff
@@ -1,29 +1,32 @@
-/** @odoo-module alias=mass_mailing.wysiwyg **/
+/** @odoo-module **/
 
-import Wysiwyg from "web_editor.wysiwyg";
-import MassMailingSnippetsMenu from "mass_mailing.snippets.editor";
-import {closestElement} from "@web_editor/js/editor/odoo-editor/src/OdooEditor";
-import Toolbar from "web_editor.toolbar";
+import { ComponentWrapper } from "web.OwlCompatibility";
+import { Wysiwyg } from "@web_editor/js/wysiwyg/wysiwyg";
+import { closestElement } from "@web_editor/js/editor/odoo-editor/src/OdooEditor";
+import { Toolbar } from "@web_editor/js/editor/toolbar";
+import { requireWysiwygLegacyModule } from "@web_editor/js/frontend/loader";
+import "@web_editor/js/wysiwyg/wysiwyg_iframe";
 
-const MassMailingWysiwyg = Wysiwyg.extend({
+export class MassMailingWysiwyg extends Wysiwyg {
     //--------------------------------------------------------------------------
     // Public
     //--------------------------------------------------------------------------
 
-    startEdition: async function () {
-        const res = await this._super(...arguments);
+    async startEdition() {
+        const res = await super.startEdition(...arguments);
         // Prevent selection change outside of snippets.
         this.$editable.on('mousedown', e => {
             if ($(e.target).is('.o_editable:empty') || e.target.querySelector('.o_editable')) {
                 e.preventDefault();
             }
         });
-        this.snippetsMenuToolbar = this.toolbar;
+        this.snippetsMenuToolbarEl = this.toolbarEl;
+        this.snippetsMenuToolbar$El = this.toolbar$El;
```

Comment:
Well I'm in the camp of those who think the widget should be the element itself, like web components. Therefore, in my mind it serves no purpose to suffix with `El` since the widget _is_ the element and vice versa. Unfortunately (for me), this is not the case in Odoo and therefore I agree that the distinction does make sense to some degree, yes.